manage.py

```diff
@@ -111,6 +111,14 @@ def parse_args():
     startup = importlib.import_module(edx_args.startup)
     startup.run()
 
+    # Prevent django-nose from passing --collect to nosetests
+    # while still leaving it on sys.argv so that runserver sees it during
+    # restart
+    from django_nose.runner import BaseRunner
+    BaseRunner.django_opts.append('--contracts')
```

Comment:
so django_opts is an *exclusion* list? I know you didn't write it, but that's confusing when reading it.